Develop (#39)
DX-DeveloperExperience/git-webhooks: docs/guide/gettingStarted.md

If you just want to test our project, without cloning it, you can run a container with one of the existing versions in [DockerHub]().

```

```

## Github/Gitlab configuration

### Add a webhook

> Once the application is running, you can configure the webhook attachs to your git repository with the url : `http://localhost:3000/webhook`. You can select only the events you want to receive, or select all of them.

Reviewer: ```suggestion
Once the API is running, you can configure create a webhook to your git repository with the url : `http://l<url of your server>/webhook`. You can also select the events you want to receive, or select all of them.
```

> You can use [ngrok](https://ngrok.com/) to convert localhost url to public url.

#### Github

You can add as many webhooks as you want. Just go to your repository settings: `https://github.com/:owner/:repo/settings/hooks`, add click the `Add webhook` button.

Now you can configure the `Payload URL`, select the `send me everything` option and save this configuration.

#### Gitlab

Go to your repository integrations settings: `https://gitlab.com/:owner/:repo/settings/integrations`, configure the webhook URL and select all the events you want to intercept. Finally, save it via the `Add webhook` button.

### Add token access

> This project allow you to interact with Github and Gitlab repositories. You can update commit status, add comments on issues or Pull Request/ Merge Request, etc.

Reviewer: ```suggestion
This project allows you to interact with Github and Gitlab repositories. You can update commit status, add comments on issues or Pull Request/ Merge Request, etc.
```
